cmd/flarectl/flarectl.go

```diff
@@ -14,6 +14,13 @@ func main() {
 	app.Name = "flarectl"
 	app.Usage = "Cloudflare CLI"
 	app.Version = "2017.10.0"
+	app.Flags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "orgid",
+			Usage: "Optional organization ID",
+			Value: "",
```

Comment:
Instead of only relying on a flag, have you considered using `EnvVar` to capture this value too?

```suggestion
			Value: "",
			EnvVar: "CF_ORG_ID"
```

Reply:
This is really a nice addition